Sources/SystemNotification/SystemNotificationContext.swift

```diff
@@ -74,6 +74,15 @@ public extension SystemNotificationContext {
     ) {
         present(message, afterDelay: delay)
     }
+	
+	public func presentStickyMessage<IconType: View>(
```

Comment:
The indentation looks a bit off, can you take a look?

Reply:
This code should **not** have been included in the PR. 
Will fix. Thanks